packages/css-prettifier/src/index.js

```diff
@@ -0,0 +1,53 @@
+import memoize from '@emotion/memoize'
+import { compile, serialize, combine, tokenize } from '@emotion/stylis'
+
+// adjusted https://github.com/thysultan/stylis.js/blob/68b9043427c177b95a0fd6a2a13f5b636bf80236/src/Serializer.js#L26-L34
```

Comment:
I've started by forking the original `stringify` because I thought I would only have to change a thing or two... but I had to change a little bit more than that, somewhat to the point that it might not be worth it and maybe it would be better to just implement a specific to this use case `serialize`, rather than acting like its a Stylis plugin. OTOH - this works now and Im not sure if I want to refactor it.